tests/controllers/test_collection_instrument.py

```diff
@@ -230,6 +230,21 @@ def test_get_instrument_by_incorrect_id(self):
         # Then that instrument is not found
         self.assertEqual(instrument, None)
 
+    def test_remove_only_eq_not_seft_cis(self):
+        # Given there is an instrument in the db for a SEFT
+        # And an eQ is added to collection exercise id
+        self._add_instrument_data(ci_type="EQ")
+
+        # Then the user unselects this eQ and exercise instrument is updated
+        self.collection_instrument.update_exercise_eq_instruments("db0711c3-0ac8-41d3-ae0e-567e5ea1ef87", [])
```

Comment:
You can use the env COLLECTION_EXERCISE_ID

Reply:
Updated